test/NHSD.BuyingCatalogue.Documents.API.Integration.Tests/Features/Get Solution Documents.3657.feature

```diff
@@ -28,11 +28,11 @@ Scenario Outline: 1. Correct file names are returned for a solution
 		| <FileNames> |
 
 	Examples:
-		| SolutionId | FileNames                                                                          |
-		| sol1       | File1.txt, File2.txt                                                               |
-		| sol2       | File1.txt, File.ouroboros                                                          |
-		| sol3       | File.ouroboros                                                                     |
-		| sol4       | File                                                                               |
-		| sol5       | File with whitespace in its name                                                   |
+		| SolutionId | FileNames                                                               |
+		| sol1       | File1.txt, File2.txt                                                    |
+		| sol2       | File1.txt, File.ouroboros                                               |
+		| sol3       | File.ouroboros                                                          |
+		| sol4       | File                                                                    |
+		| sol5       | File with whitespace in its name                                        |
 		| sol6       | děkuji.pdf,Благодаря ти.blin,ありがとうございました.txt,תודה.jpeg, شكرا لكم.png, 谢 |
-		| sol99      |                                                                                    |
+		| sol99      |                                                                         |
```

Comment:
I think the vertical bars might have looked better where they were (closer in line to the bar for sol6).

Reply:
👍